### PR TITLE
Troubleshooting

### PR DESCRIPTION
**Pull request by mcwimm**
Added a section for troubleshooting advise.

@jvollhueter @rbnmj @jack-w-hill  @ChrisWudel Do you remember any other problems you had when you first ran pyManga?

We could add also a section about OGS. @jvollhueter do you remember the issue Emma had?

### CODE REVIEW

**Comment by rbnmj** [comment]
I couldn't find major issues with the installation guide. The main problem I had when installing seems to be updated already (missing file path for running OGS example).  
<br>
However, I do have some minor suggestions I could implement if you don't mind:  
**Install Python and Python Modules**  
* Module requirements on website do not match requirements.txt file in pymanga root. 
  * I think scipy is needed for OGS and is neither listed on website nor in the txt (is this correct?).
  * Requirements.txt includes: pytest, numpy, lxml, matplotlib, pandas, pytest-subtests, pdoc, and vtk == 9.2.2
**Installation of Python Modules in Windows**  
* Incomplete module list in module installation section (see above).
* vtk version has to be added (`py -3.7 -m pip intall -v "vtk==9.2.2`).
**pyMANGA execution in Windows**  
* Right clicking folder to open cmd should be excluded since it is not working by default (on win 10).
  * Instructions on how to add right-click cmd could then be removed to avoid confusion.
* `py MANGA.py -h` should be changed to `py -3.7 ...` to prevent problems (this is already done for **Fist applications**).
* 
<br>

**First applications/Setup with OGS/First applications of pyMANGA with OGS in Windows**
* Change OGS version 6.3.0 to 6.4.0?

<br>

* Figure descriptions are missing in all sections.

**Comment by rbnmj** [comment]
Ubuntu sections are mostly untouched for now. I will update Ubuntu sections when Windows sections are done.

**Review by mcwimm — COMMENTED** [review]
Looks good! It can be merged if the comments have been addressed.
I can`t approve this PR as I`m the creator. So you need to approve it ;)

File: web/content/en/docs/getting_started/First_applications_of_pyMANGA.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ This is especially recommended for beginners who have little experience with **P
...
+## 1 Setup without OpenGeoSys
 
 > Here, we explain how to run a simple pyMANGA setup
 > In this setup, 10 black mangrove trees grow at seawater salinity. 
 > Tree growth follows the BETTINA approach.
 > Model output is written to the folder 'ModelOutput' in the same directory.  
-> More examples can be found [here](/docs/benchmarks/) and in our [benchmark gallery](https://github.com/jbathmann/pyMANGA/tree/master/Benchmarks/ModuleBenchmarks/).
+> More examples can be found [here](/docs/benchmarks/) and in our [benchmark gallery](https://github.com/pymanga/pyMANGA/tree/master/Benchmarks/ModuleBenchmarks/).
 
 First you have to use the console interface again to navigate to the file location of the **pyMANGA** main level.
```
[the remark this answers is not in the document]
Comment (rbnmj) [reply web/content/en/docs/getting_started/First_applications_of_pyMANGA.md:29]: I agree. I think this is already added to "Installation" anyway.
I don't mind leaving the Ubuntu help in. After cutting the first part (manga.py -h) it's only two lines for Ubuntu.

**Comment by mcwimm** [comment]
imo, this is ready to be merged. @rbnmj can you check/approve/merge?

**Review by rbnmj — APPROVED** [review]

**Pull request merged**